syntax = "proto3";

package sample;

// 서비스 정의
service SampleService {
  rpc SayHello (SampleRequest) returns (SampleResponse);
}

// 메시지 정의
message SampleRequest {
  string name = 1;
}

message SampleResponse {
  string message = 1;
}